### ISSUE TITLE
TypeError: Cannot read property 'text' of undefined

### ISSUE THREAD

**Issue by christian110011** [json-schema-form/angular-schema-form#624]
any idea what the reason is for this error?
thank you 
Chris

TypeError: Cannot read property 'text' of undefined
    at http://localhost:8089/bundle.js:60554:5029
    at Array.reduce (native)
    at l (http://localhost:8089/bundle.js:60554:4979)
    at Object.build (http://localhost:8089/bundle.js:60554:5710)
    at g (http://localhost:8089/bundle.js:60554:28548)
    at y (http://localhost:8089/bundle.js:60554:28129)
    at Object.<anonymous> (http://localhost:8089/bundle.js:60554:29011)
    at Scope.$digest (http://localhost:8089/bundle.js:15920:41)
    at http://localhost:8089/bundle.js:16123:27
    at completeOutstandingRequest (http://localhost:8089/bundle.js:5584:11)(anonymous function) @ angular.js:12520(anonymous function) @ angular.js:9292Scope.$digest @ angular.js:15914(anonymous function) @ angular.js:16091completeOutstandingRequest @ angular.js:5552(anonymous function) @ angular.js:5829

**Comment by nicklasb** [comment]
This could literally be anything. Please create a plunkr or jsfiddle example.

**Comment by jdpnielsen** [comment]
I ran into the same error message when trying to install via npm and bundle with browserify. 
I fixed it by including the bootstrap decorator.

See [where is bootstrap decorator](https://github.com/Textalk/angular-schema-form/blob/development/dist/WHERE_IS_BOOTSTRAP_DECORATOR.md).

If you are using a similar setup as me:
`npm install angular-schema-form-bootstrap --save`

and then something like:

```js
'use strict';

var angular = require('angular');

//Include modules
require('angular-schema-form');
require('angular-schema-form-bootstrap');

// Inject dependencies
angular.module('myWebApp', ['schemaForm']);

```
